### PR TITLE
feat(style): experimental style adjustment

### PR DESCRIPTION
**Pull request by Muthaias**
Adjusts the styling:
* note that the biggest size difference is in mobile landscape
* use `vh` in most places in order to make the sizes consistent
* remove colors from value bars
* remove rounded corners from value bars
* increase size of value bars
* adjust styling of icon container
* group stats content tighter together

Depending on preference it might seem a bit small on mobile portrait. I considered keeping it as it was but preferred consistency in sizing rather than a larger mobile portrait. I would argue that the game should not be played in mobile portrait. :P

To test:
* checkout and run
* verify that the game can be played in different sizes
* preferrably test on actual mobile device

### CODE REVIEW

**Comment by Muthaias** [comment]
Screen Shot 2020-10-21 at 23 34 29
Intended look

**Review by Greenheart — CHANGES_REQUESTED** [review]
I've tested this on multiple devices and given this some thought.

## Really nice work to clarify and simplify the UI! Some really nice UI/UX improvements including:
* Much clearer stats UI with clean colors, shadows and the leaf shapes.
* The green header and new background color feels much more polished than the previous ones.
* Greatly improved support for landscape mode on phones (although we should still recommend players to use portrait mode to get the best experience)
* Cards feel lighter to swipe on desktop, which is a nice UX improvement for testing (and desktop players)

I like the resulting UI and UX since this is a clear step forward. However, there are some issues with the implementation that I'd like to iterate further upon.

## Potential improvements

### 1. UI bug likely caused by `vh` units on mobile
* `vh` units on mobile are tricky since they don't take the mobile browser UI (address bar, potential on screen navigation buttons for android) into account. Instead, `vh` units are based around the viewport itself rather than the actually usable viewport, like it behaves on desktop browsers. In our case, this causes some issues when testing on my primary phone.
sff-oneplus-6s

A potential fix might be to use `px` values instead, and potentially switch between a smaller value for mobile and a larger for desktop.

The `40vh` within cardSize would translate to 240px (`0.4 * 600px = 240px`)

Then we could use `400px` as the base value for larger screens.

### 2. Replace `vh` units in `Stats.tsx` to make sure the stats work on mobile screens (see screenshot above)

Maybe we could reuse the spacing scale from `CardView.tsx` to still allow switching between desktop and mobile modes?

### 3. Simplify styled component selectors
Finally, a quick improvement I noticed when testing is that we could remove all the `& >` uses in selectors in `CardView.tsx`. Nested classes work get the same result with less code.

**Comment by Muthaias** [comment]
I agree on the issue fixes. I am pretty sure they are related to me using gap in a flex layout. It seems to not be supported very well. So I'll look into a more stable solution. Padding for example.

I am very sceptical towards reducing the specificity of the selectors to save bytes though. They do serve a purpose and changing it will make it more complex to add sub-elements that are not intended to be styled by the component. (I am not saying this is a great component structure at the moment but I am not a fan of making it worse)

To summarize. I'll currently look into 1 and 2.

**Comment by Muthaias** [comment]
I have added a fix for the issue with the stats. I hope you don't mind reconsidering about `3`.

**Comment by Greenheart** [comment]
Thanks for fixing the layout! You were right about the issue with gap in flex layouts. It's a very recent addition and I was testing in the latest Ecosia browser for Android (based on an older version of Chromium).

Let's ignore the third point until a future UI update.

Looking good - merging! 🌍✨

**Review by Greenheart — APPROVED** [review]
